### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md`

### PR DESCRIPTION
**Pull request by zhiquanchi**
Added Chinese page for Dependencies with yield
添加了Dependencies with yield的中文页面

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8081cffca068fa2dfb9c26b5549740df6c23cc52 at: https://317ebf21.fastapitiangolo.pages.dev

File: docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    都可以作为 **FastAPI** 的依赖项。
+
+    实际上，FastAPI内部就使用了这两个装饰器。
+
+
+## 使用 `yield` 的数据库依赖项
+
+例如，您可以使用这种方式创建一个数据库会话，并在完成后关闭它。
+
+在发送响应之前，只会执行 "yield "语句及之前的代码：
```
[the remark this answers is not in the document]
Comment (zhiquanchi) [reply docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md:26]: 我将会修改它

File: docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    {!> ../../../docs_src/dependencies/tutorial008_an_py39.py!}
+    ```
+
+=== "Python 3.6+"
+
+    ```Python hl_lines="5  13  21"
+    {!> ../../../docs_src/dependencies/tutorial008_an.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
```
[the remark this answers is not in the document]
Comment (zhiquanchi) [reply docs/zh/docs/tutorial/dependencies/dependencies-with-yield.md:83]: 我将会把Python的版本和en文档同步

**Comment by tiangolo** [comment]
Please let me know when this is ready @zhiquanchi 🤓 ☕

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5954a06cf1c3494de63a755ef67a054409b2c22b at: https://f3f63204.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Thanks for the interest! @zhiquanchi 🚀 

Do you think this one is ready with your suggested changes applied? @Aruelius 🤓

**Comment by Aruelius** [comment]
> Thanks for the interest! @zhiquanchi 🚀
> 
> Do you think this one is ready with your suggested changes applied? @Aruelius 🤓

Hi, maybe you should replace all the `Python 3.6` to `Python 3.8`.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 2f6b0a225a7bd414a6074dfd3a67121c21bdaeca at: https://6ea02cba.fastapitiangolo.pages.dev

**Review by Aruelius — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for your help! @Aruelius @zhiquanchi 🚀✨

**Pull request merged**